app/battle_preparation/knight.py

```diff
@@ -0,0 +1,24 @@
+class Knight:
+    def __init__(self, name: str, power: int, hp: int) -> None:
+        self.name = name
+        self.power = power
+        self.hp = hp
+        self.protection = 0
+        self.equipment = None
```

Comment:
do you set this attribute anywhere?

Reply:
Yes, in the main module. Protection for calculating armour's effect and equipment just for info